test/hot/hot-flag.test.js

```diff
@@ -19,7 +19,7 @@ describe('--hot flag', () => {
     it('should warn when --hot and --no-hot both are passed', (done) => {
         const { stderr, stdout } = run(__dirname, ['--no-hot', '--hot']);
         expect(stderr).toContain(
-            '[webpack-cli] You provided both --hot and --no-hot. We will use only the last of these flags that you provided in your CLI arguments',
+            'You provided both --hot and --no-hot. We will use only the last of these flags that you provided in your CLI arguments',
```

Comment:
WE should keep `[webpack-cli]` other loaders/plugins/etc also can output values, it is very hard to detect who it output is

Reply:
I will update